### ISSUE TITLE
Error Message - DIM can't store data

### ISSUE THREAD

**Issue by matt513** [DestinyItemManager/DIM#10776]
### DIM Version

Version 8.44.0 (release), built on 10/27/2024, 7:23:40 PM

### Browser Details

Firefox 132.0

### OS Details

Windows 10

### Describe the bug

Upon loading into the Inventory page of DIM, I am receiving a red message at the top right stating "DIM can't store data". I've tried logging in and out as well as clearing cookies and site data for DIM. 

I also saw an other person having a similar issue using Zen Browser on reddit: https://www.reddit.com/r/zen_browser/comments/1gfps48/app_cant_store_data_in_browser/

![image](https://github.com/user-attachments/assets/ce513e7f-ba0c-47ae-a26e-5e5eea87faf7)

### Logs

```shell
Experiment add-an-image-to-pdf-with-alt-text-rollout has unknown featureId: addAnImageInPDF RemoteSettingsExperimentLoader.sys.mjs:709:21
Experiment tab-hover-preview-release-rollout has unknown featureId: tabPreview RemoteSettingsExperimentLoader.sys.mjs:709:21
TypeError: linkMap is undefined
    Links_onLinkChanged resource://gre/modules/NewTabUtils.sys.mjs:2003
    PlacesProvider__callObservers resource://gre/modules/NewTabUtils.sys.mjs:697
    handlePlacesEvents resource://gre/modules/NewTabUtils.sys.mjs:671
NewTabUtils.sys.mjs:699:19
Could not write session state file  DOMException: Could not move `C:\Users\matt5\AppData\Roaming\Mozilla\Firefox\Profiles\f7mww221.default-release\sessionstore.jsonlz4' to `C:\Users\matt5\AppData\Roaming\Mozilla\Firefox\Profiles\f7mww221.default-release\sessionstore-backups\previous.jsonlz4': source file does not exist (NS_ERROR_FILE_NOT_FOUND) <empty string> SessionFile.sys.mjs:449:17
WindowsJumpLists: Failed to fetch favicon for  https://github.com/login NS_ERROR_FAILURE: WindowsJumpLists.sys.mjs:257:29
WindowsJumpLists: Failed to fetch favicon for  https://github.com/DestinyItemManager/DIM/wiki/Troubleshooting NS_ERROR_FAILURE: WindowsJumpLists.sys.mjs:257:29
WindowsJumpLists: Failed to fetch favicon for  https://github.com/password_reset/ACAF4X2IYCQ52XIIRXTLZ5LHEMFUNBNFMVWWC2LMWJWWC5DUGUYTG2CAM5WWC2LMFZRW63NFMZXXEY3FYKZXI53PL5TGCY3UN5ZF65TFOJUWM2LFMTBLC5DXN5PWMYLDORXXEX3NMV2GQ33EURXG63TFXJTG64TDMVSF653FMFVV64DBONZXO33SMRPXEZLTMV2ME?auto=true NS_ERROR_FAILURE: WindowsJumpLists.sys.mjs:257:29
Partitioned cookie or storage access was provided to “https://www.googletagmanager.com/static/service_worker/4al0/sw_iframe.html?origin=https%3A%2F%2Fwww.bungie.net” because it is loaded in the third-party context and dynamic state partitioning is enabled.

Request to access cookie or storage on “https://10149754.fls.doubleclick.net/activityi;src=10149754;type=bungap;cat=buapvi;ord=7817363467558;npa=0;auiddc=2021647449.1729104671;u3=https%3A%2F%2Fwww.bungie.net%2Fen%2FUser%2FJoinUp;pscdl=noapi;frm=0;gtm=45fe4as0h1v9181795285z8812098552za201zb812098552;gcd=13l3l3l3l1l1;dma=0;tag_exp=101533421~101823848~101878899~101878944~101925629;epver=2;~oref=https%3A%2F%2Fwww.bungie.net%2Fen%2FUser%2FJoinUp%3Fclient_id%3D13672%26state%3D2afe6c23-73d5-4c33-8aa5-764d5755a6af%26response_type%3Dcode%26bru%3D%252Fen%252FOAuth%252FAuthorize%253Fclient_id%253D13672%2526response_type%253Dcode%2526state%253D2afe6c23-73d5-4c33-8aa5-764d5755a6af?” was blocked because it came from a tracker and content blocking is enabled.

The resource at “https://images.opencollective.com/michabbb/da16995/logo.png?height=130” was blocked by OpaqueResponseBlocking. Reason: “after sniff: status code is not in allowed range”.
WindowsJumpLists: Failed to fetch favicon for  https://github.com/login NS_ERROR_FAILURE: WindowsJumpLists.sys.mjs:257:29
WindowsJumpLists: Failed to fetch favicon for  https://github.com/DestinyItemManager/DIM/wiki/Troubleshooting NS_ERROR_FAILURE:
```

**Comment by bhollis** [comment]
Duplicate of #10764
